docs/source/plugins.rst

```diff
@@ -5,30 +5,33 @@
 Plugins
 =======
 
-Plugins that come with the **IBM z/OS core collection** augment Ansible's core
+Plugins that come with the **IBM z/OS core collection** compliment Ansible's core
```

Comment:
compliment with i = "says something nice about"
complement with e = "improves upon something else"
I think you need the 'e' here... it just jumped out at me.